attestation-service/verifier/src/cca/mod.rs

```diff
@@ -118,6 +135,8 @@ impl Verifier for CCA {
             .context("decrypt the ear with the decoding key")?;
 
         let ear_nonce = plain_ear.nonce.context("get nonce from ear")?;
+        let evidence = &plain_ear.submods["CCA_SSD_PLATFORM"].annotated_evidence;
```

Comment:
Is submods expected to have a field named "CCA_SSD_PLATFORM"? BTreeMap implementation shows that if we use `[...]`, the key should exist, or the process would panic https://doc.rust-lang.org/src/alloc/collections/btree/map.rs.html#2328

Reply:
Ok. I get your point that veraison is expected to have such a field. However, from the perspective of AS we cannot expect it will happen as a malicious/wrong remote veraison might do unexpected things. In such case, the caller of `verifier` should know why the function fails to execute rather than unexpected panic. That is to say we'd better use an error handling here.  wdyt?

Reply:
make sense, will update.

Reply:
fixed.